inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/DataSeparator.java

```diff
@@ -29,7 +29,8 @@ public enum DataSeparator {
     DASH("-", 45),
     SOH("\001", 1),
     STX("\002", 2),
-    ETX("\003", 3);
+    ETX("\003", 3),
```

Comment:
We should support custom separators for users, not need to define them in an Enum class.

Reply:
I just removed `DataSeparator`